### ISSUE TITLE
defined(@array) is deprecated

### ISSUE THREAD

**Issue by strk** [linz/linz-bde-uploader#49]
Running the following test command from the `bde-processor-deploy` Vagrant machine results in a couple of warnings about deprecated construct:

The command:
```
linz_bde_uploader -verbose -c /etc/linz-bde-uploader/linz_bde_uploader.conf -full
```

The warnings:
```
defined(@array) is deprecated at /usr/share/perl5/Mail/Sender.pm line 318.
        (Maybe you should just omit the defined()?)
defined(@array) is deprecated at /usr/share/perl5/Mail/Sender.pm line 2693.
        (Maybe you should just omit the defined()?)
```

**Comment by strk** [comment]
The Vagrant machine has perl v5.18.2 installed

**Comment by palmerj** [comment]
This issue can only be solved by updating libmail-sender-perl which supports Mail::Sender 0.8.23

**Comment by palmerj** [comment]
This can be fixed by upgraded to Ubuntu 18.04
